### ISSUE TITLE
Why is this amazing package still on version 0.0.1?

### ISSUE THREAD

**Issue by mrismailt** [moodymudskipper/typed#25]
First of all, I'm a massive fan of this package. You have brought something much needed to R. 

I've been using this package for a while on various projects, but I never realized that {typed} is still only on version 0.0.1?

We are slowly trying to adopt {typed} into an extensive production R system, and this fact has raised some concerns with the team. We wanted to hear from you directly what the reason for this is and if we should be concerned about the maturity of the package.

Thanks for all your great work and fantastic contribution to the R community!

**Comment by moodymudskipper** [comment]
Hi @mrismailt and thanks so much for the kind words. I'm glad that my work is being used and appreciated.

I've done the most part of the work on {typed} some time ago and have not been working much since then.
I still consider {typed} a somewhat experimental package because I have not received a lot of feedback of users and I don't use it much myself.

If there is demand for bug fixes and features I'm happy to spend time on it though. I see there are a handful of small issues that might justify a new patch update.

So in short : 

* It is indeed not under active development

* But I'm available for support

* I'm happy to jump back in for more extensive features at some point if I see demand

**Comment by mrismailt** [comment]
Thank you for your quick response, @moodymudskipper. It is somewhat disconcerting to discover that such a fantastic package is not under active development (though reassuring that you are available for support). In this case, my follow-up question would be whether you have recommendations for another package that solves the same problem (implementing a type system in R) that _is_ under active development and more mature for production use-cases? Thanks!

**Comment by moodymudskipper** [comment]
May I ask what you and your team would expect from further development?

Depending on your needs maybe some of the packages I mention at the end of the README would help.

**Comment by pedrocava** [comment]
Dropping by to say we use `typed` in production at my company, and although we don't have needs or expectations for new features, it is reassuring to hear you'll be around to help maintain it @moodymudskipper.

**Comment by moodymudskipper** [comment]
It's really cool to know it's being used in production, thanks so much @pedrocava.

**Comment by phineas-pta** [comment]
eventhough i just found out this package, i'm excited to know that it is still maintained 😍, others typing packages are completely halted, the R community seems to have no interest in static typing 😞, meanwhile Python now has built-in typing

**Comment by balthasars** [comment]
Also thanks from me to @moodymudskipper for the maintenance and creating a useful implementation of something for which there is a dire need within the R.

I believe it would inspire confidence in potential users to see some examples of packages that use {typed} in its implementation (perhaps links at the end of the README file) or on a pkgdown page for this project. Moreover for bringing it to more users, it might help to mention that {typed} forms part of [the rpp project](https://blog.q-lang.org/posts/2021-10-16-project/).

**Comment by latot** [comment]
Me too! I'm trying to implement this on pruduction features!

Thx @moodymudskipper for this great package!

**Comment by moodymudskipper** [comment]
In the end I've not been very responsible in maintaining this package, I apologise for those that were affected, especially since some bugs compromise some use cases. 
I don't want to overpromise from now on but in 2025 I'll have some more structured time for OS, I want to cleanup the bugs and ship this to CRAN, but I don't plan on implementing new features.

I'm closing this to cleanup the issue tracker but thanks for your feedback and input and feel free to discuss further here or in new tickets.
